lib/IDE/IDERequests.cpp

```diff
@@ -375,7 +375,25 @@ bool CursorInfoResolver::visitCallArgName(Identifier Name,
   if (isa<ModuleDecl>(D))
     return true;
 
-  bool Found = tryResolve(D, nullptr, nullptr, Range.getStart(), /*IsRef=*/true);
+  if (swift::ide::resolveArgName(
+          D, Name, Range, [this, Name, Range](ParamDecl *param) {
+            bool isExtArgName = param->getArgumentNameLoc().isValid();
+            bool Found = param->getArgumentName() == Name &&
+                         !swift::ide::isParamFromOverriddenConstructor(param) &&
```

Comment:
Why do we need to check for `isParamFromOverriddenConstructor` here? Could you add a comment that describes the reason?